selfdrive/car/toyota/tunes.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+from selfdrive.car.toyota.values import MIN_ACC_SPEED, PEDAL_HYST_GAP
+
+
+
+###### LONG ######
+def set_long_tune(tune, name):
+  if name == 'PEDAL':
```

Comment:
use an enum for the tunes https://docs.python.org/3/library/enum.html

Reply:
Done